medicines/search/definitions/indexes/default.json

```diff
@@ -244,7 +244,7 @@
       "filterable": false,
       "key": true,
       "retrievable": true,
-      "searchable": false,
+      "searchable": true,
```

Comment:
This is here because we need to make the `metadata_storage_name` searchable if we're going to be able to uniquely identify index records by the blob name. The search indexes will need to be updated in all environments after this PR is merged.

Reply:
yes, saw this in action on mob call 👍